tests/test_token_builder.cpp

```diff
@@ -54,6 +54,21 @@ SingleTokenWithSymbol) {
 	EXPECT_EQ(tokenizer.get_tokens()[1]->get_pattern(), "abc");
 }
 
+TEST_F(TestTokenBuilder,
+SingleTokenWithDescription) {
+	TokenBuilder<int> tb(&grammar, &tokenizer, "abc");
+	tb.symbol("ABC");
+	tb.description("abc token");
+	tb.done();
```

Comment:
I would personally like if you created one more token here without description and then in test compared that description is equal to name.

Reply:
Another token 'DEF' has been added and the test was renamed to MultipleTokensDescription. Is it ok?